pkg/controlsvc/controlsvc.go

```diff
@@ -274,6 +274,7 @@ func (s *Server) RunControlSession(conn net.Conn) {
 		if len(cmdBytes) == 0 {
 			continue
 		}
+		s.nc.GetLogger().Debug("Control service received command: %s\n", string(cmdBytes))
```

Comment:
We will need to sanitise this to prevent Log Injection Attacks i think. 
I have already created `SanitizedDebug` to do exactly this, however it comes with a performance cost :smile: